README.md

```diff
@@ -1,71 +1,96 @@
-# Prometheus SCW Service Discovery
+A service discovery for the [Scaleway](https://www.scaleway.com/) cloud platform compatible with [Prometheus](https://prometheus.io).
 
-Prometheus SCW Service Discovery converts your Scaleway server list and converts to Prometheus targets.
+Credits to [Simon Pasquier](https://github.com/simonpasquier) who wrotes most of the code on hiw own repository before merging it into this one.
 
-This project is adapted from the official [Consul SD](https://github.com/prometheus/prometheus/tree/master/documentation/examples/custom-sd).
+## How it works
 
-An official Prometheus [blog post](https://prometheus.io/blog/2018/07/05/implementing-custom-sd/) explains how to write a Custom Service Discovery.
+This service gets the list of servers from the Scaleway API and generates a file which is compatible with the Prometheus `file_sd` mechanism.
 
-## Usage
+## Pre-requisites
 
-Download the binary from the last release, corresponding to your own architecture.
+You need your Scaleway secret key (token). You can create this token [in the console](https://cloud.scaleway.com/#/credentials).
 
-Help:
-```
-./prometheus-scw-sd -h
-```
+## Installing it
 
-Start the discoverer:
-```
-./prometheus-scw-sd --token="$TOKEN" --output.file="prometheus-scw-sd.json"
-```
+Download the binary from the [Releases](https://github.com/simonpasquier/prometheus-scaleway-sd/releases) page.
+
+## Running it
 
-Using servers private IP, custom port and time interval in second:
 ```
-./prometheus-scw-sd \
-    --token="$TOKEN"                       \
-    --output.file="prometheus-scw-sd.json" \
-    --time.interval="90"                   \
-    --port="1234"                          \
-    --private
+usage: sd adapter usage --scw.token=SCW.TOKEN [<flags>]
```

Comment:
See comment below regarding `--scw.token`

Reply:
changed in 739b856